.travis.yml

```diff
@@ -1,17 +1,20 @@
-sudo: true
-
+os: linux
+dist: xenial
 language: cpp
 
 services:
   - docker
 
 before_install:
-  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/install-package.sh
-  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/master/ci/build-package.sh
+  - openssl aes-256-cbc -K $encrypted_3959f4b55449_key -iv $encrypted_3959f4b55449_iv
+    -in ./deploy_key.enc -out ./deploy_key -d
+  - export SSH_KEY_PRIVATE=$(cat deploy_key)
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/fix/ci/ci/install-package.sh
+  - wget https://raw.githubusercontent.com/tue-robotics/tue-env/fix/ci/ci/build-package.sh
   - export PACKAGE=${TRAVIS_REPO_SLUG#*/}
 
 install:
-  - bash install-package.sh --package=$PACKAGE --branch=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH} --commit=$TRAVIS_COMMIT --pullrequest=$TRAVIS_PULL_REQUEST
+  - bash install-package.sh --package=$PACKAGE --branch=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH} --commit=$TRAVIS_COMMIT --pullrequest=$TRAVIS_PULL_REQUEST -i="tuerobotics/tue-env" -s="true" -sk="$SSH_KEY_PRIVATE"
```

Comment:
Don't like the short version of the arguments, Not readable at all

Reply:
Done.